Sources/Filters.swift

```diff
@@ -89,6 +89,11 @@ struct StringFilters {
     }
   }
 
+  static func camelToSnakeCase(_ value: Any?) throws -> Any? {
+    guard let string = value as? String else { throw FilterError.invalidInputType }
+    return try snakecase(string).lowercased()
```

Comment:
I haven't seen it either, I was just wondering what the correct / default behaviour should be. Looking at some libraries in other languages, the default seems to be lowercase, so let's do that 👍 

You could always add a parameter to the filter to disable the lowercasing (see http://stencil.fuller.li/en/latest/builtins.html#built-in-filters).